tests/commands/test-sdk.js

```diff
@@ -0,0 +1,630 @@
+					}
+					logger.buffer.should.startWith(`SDK Install Locations:\n   ${sdksPath.cyan}${' [default]'.grey}\n\nInstalled SDKs:\n   ${('9.2.0.GA').cyan}${' [selected]'.grey}  ${'9.2.0.v20200923092031  '.magenta}${sdkPath}\n\n`);
+					logger.calls[0].should.eql([ 'banner', undefined ]);
+					logger.calls[1].should.eql([ 'log', 'SDK Install Locations:' ]);
+					logger.calls[2].should.eql([ 'log', `   ${sdksPath.cyan}${' [default]'.grey}` ]);
+					logger.calls[3].should.eql([ 'log', undefined ]);
+					logger.calls[4].should.eql([ 'log', 'Installed SDKs:' ]);
+					logger.calls[5].should.eql([ 'log', `   ${('9.2.0.GA').cyan}${' [selected]'.grey}  ${'9.2.0.v20200923092031  '.magenta}${sdkPath}` ]);
+					logger.calls[6].should.eql([ 'log', undefined ]);
+					// Release listing header
+					logger.calls[7].should.eql([ 'log', 'Releases:' ]);
+					// Verify 9.2.0.GA is listed and thinks it is installed
+					logger.buffer.should.containEql(`   ${'9.2.0.GA'.cyan} [installed]`);
+
+					finished();
+				});
+			});
+		});
+
+		describe('install', () => {
+			// TODO: Test with install location in place we don't have rights to create/write
+			it('with pre-existing sdk, without --force', function (finished) {
+				const cli = {
+					argv: {
+						_: [ 'install' ],
+						$: 'titanium',
+						default: false,
+						force: false,
+						quiet: false,
+						'progress-bars': false,
+						version: '9.2.0.GA'
+					},
+					env: {
+						installPath: sdksPath,
+						os: {
+							name: 'osx',
+							sdkPaths: [ sdksPath ],
+						},
+						sdks: {
+							'9.2.0.GA': {
+								manifest: {
+									name: '9.2.0.v20200923092031',
+									version: '9.2.0.GA',
+								},
+								path: path.join(sdksPath, 'mobilesdk/osx/9.2.0.GA')
+							}
+						}
+					},
+					addAnalyticsEvent: (name, obj) => {}
+				};
+
+				SDK.run(logger, config, cli, function (err) {
+					if (err) {
+						return finished(err);
+					}
+
+					logger.calls[0].should.eql([ 'banner', undefined ]);
+					logger.calls[1].should.eql([ 'log', `Titanium SDK ${'9.2.0.GA'.cyan} is already installed!\n` ]);
+					logger.calls[2].should.eql([ 'log', `Run '${('titanium sdk install 9.2.0.GA --force').cyan}' to re-install.\n` ]);
+
+					finished();
+				});
+			});
+
+			it('with uninstalled sdk', function (finished) {
+				this.timeout(5 * 60 * 1000); // 5 minutes
+				const cli = {
+					argv: {
+						_: [ 'install' ],
+						$: 'titanium',
+						default: false,
+						force: false,
+						quiet: false,
+						'progress-bars': true,
+						version: '7.5.0.GA'
+					},
+					env: {
+						installPath: sdksPath,
+						os: {
+							name: 'osx',
+							sdkPaths: [ sdksPath ],
+						},
+						sdks: {
+							'9.2.0.GA': {
+								manifest: {
+									name: '9.2.0.v20200923092031',
+									version: '9.2.0.GA',
+								},
+								path: path.join(sdksPath, 'mobilesdk/osx/9.2.0.GA')
+							}
+						}
+					},
+					addAnalyticsEvent: (name, obj) => {}
```

Comment:
<!--
  0 failure: 
  2 warning:  tests/commands/te..., tests/commands/te...
  
  
  DangerID: danger-id-Danger;
  File: tests/commands/test-sdk.js;
  Line: 390;
-->

- :warning: tests/commands/test-sdk.js line 390 – 'name' is defined but never used. Allowed unused args must match /^_.+/u. (no-unused-vars)
- :warning: tests/commands/test-sdk.js line 390 – 'obj' is defined but never used. Allowed unused args must match /^_.+/u. (no-unused-vars)